app/views/coronavirus_form/which_goods.html.erb

```diff
@@ -0,0 +1,73 @@
+<% content_for :title do %><%= t('coronavirus_form.which_goods.title') %> - GOV.UK<% end %>
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= t('coronavirus_form.which_goods.title') %>" />
+<% end %>
+
+<div class="govuk-width-container">
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render "coronavirus_form/validation_error" %>
+        <%= form_tag({},
+          "data-module": "track-coronavirus-form",
```

Comment:
I wonder if this needs to be more specific, since we will be having multiple coronavirus forms?